addons/website_sale/views/templates.xml

```diff
@@ -814,10 +820,17 @@
     <template id="categorie_link" name="Category Link">
         <t t-if="isOffcanvas" t-set="parentCategoryId" t-valuef="offcanvas"/>
 
-        <div t-att-data-link-href="keep('/shop/category/' + slug(c), category=0)" class="form-check d-inline-block">
-            <input type="radio" t-attf-name="wsale_categories_radios_{{parentCategoryId}}" class="form-check-input pe-none" t-att-id="c.id" t-att-value="c.id" t-att-checked="'true' if c.id == category.id else None"/>
-            <label class="form-check-label fw-normal" t-att-for="c.id" t-field="c.name"/>
-        </div>
+        <a t-att-href="keep('/shop/category/' + slug(c), category=0)" class="text-decoration-none p-0 text-900">
+            <div class="form-check d-inline-block">
+                <input type="radio"
+                    t-attf-name="wsale_categories_radios_{{parentCategoryId}}"
+                    class="form-check-input pe-none "
+                    t-att-id="c.id"
+                    t-att-value="c.id"
+                    t-att-checked="'true' if c.id == category.id else None"/>
+                <label class="form-check-label fw-normal" t-field="c.name"/>
```

Comment:
Why there is no `t-att-for` for this label? :thinking:

Reply:
When a `<label>` is associated with an `<input>` using the `for` attribute, clicking the label triggers the input element's click event. This interferes with the default behavior of the `<a>` tag. In our case, when you click on the label, the input element is selected, but the routing does not occur as expected.